.circleci/config.yml

```diff
@@ -1,29 +1,40 @@
 version: 2.1
 
 executors:
-  ubuntu:
+  debian:
     docker:
-      - image: ubuntu:20.04
+      - image: debian:bullseye
```

Comment:
Why the switch to Debian?

Looking at https://computingforgeeks.com/how-to-install-docker-on-ubuntu/ we can install docker-ce pretty much the same way on Ubuntu

Reply:
Just because I use it and like it. Will switch it back to Ubuntu.